### PR TITLE
Milestone 2: User interface

### PR DESCRIPTION
**Pull request by mohapakram**
In this pull request we added:
1- A prompt for users to enter data and only mock the basic flow of the game.
2- Made sure that we can run our game without executable files and play with the basic flow.
3- Thought about all actions needed to play Tic Tac Toe.

### CODE REVIEW

**Review by adaorachi — CHANGES_REQUESTED** [review]
### Hi @mohapakram and @gurbuzertunga, :wave: 
Good job on this project. Your project runs fine. However, there are a couple of changes to be implemented in order to get your project approved. 

---

### STATUS: CHANGES REQUIRED :recycle: 

---

- [x] You already have some implementations for the game flow. But there are others that are also needed to make sure you thought of the processes or actions needed when you build the functional game. They are;
  - your game should inform the player if the selected move is invalid. 
  - your game should display board after a player's move.
  - your game should inform the players if a selected move is a winning move.
  - your game should inform the players if a selected move is a draw move.

Remember that these can be done with inline statements or `puts` statements like you have for the others. You don't have to compute any logic in them. Also, remember to call these actions repeatedly inside the loop.

- [x] Kindly avoid the use of definite loop such as calling a number to it, e.g - `9.times do ... end`,  like you have done. Loops like a `while` loop are preferable.
Remember that the purpose of this milestone is to create a mock of the game and it would not be proper calling a `9.times do ... end` on a game that will be terminated when some conditions are met inside that loop. 
Here is a hint from Odin-project on why a loop is also one of the basic requirements for the game.

- [x] In addition to the above, you should also ensure that the loop is terminated at a point. You can achieve this with a variable. For example, you can set  a variable like this - `game_on=true`, 
```
while game_on
  ...
  if condition_is_met
   ...
    game_on=false
   ...
  end
end
```

- [**optional**] It is not a mandatory requirement to check your README at this milestone, so this is considered an optional requirement but will be nice if you implement the following. Your README shows the content of a bubble_sort project. You should kindly update the content with the right information about this project or you remove the content and add just the project title, and, then update it in the final milestone. This is to ensure your README is kept decent for the time being.

---
### Kindly implement the mandatory changes above and request for another code review when you are done.
Note all requirements not marked with OPTIONAL are mandatory.

---

Should you have any question, kindly reach out to me on Slack with the username @adaorachi

Happy Coding :relaxed: 

MaryAnn Chukwuka.

File: bin/main.rb [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+  if (1..9).cover?(current_move)
+    if played_moves.include?(current_move)
+      puts 'this is an invalid move'
+      puts 'this block is aready taken'
+    else
+      played_moves.push(current_move)
+      puts "Now, your move is displayed on the board #{current_player == 1 ? user1 : user2}"
+      display_board(board)
+      counter += 1
+      if counter > 2
```
Comment (krys2fa) [inline bin/main.rb]: - [ ] [REQUIRED] You are using out-of-nowhere numbers to terminate the loop. This should not be the case.
**HINT**: It is better to have a more descriptive variable terminating the loop. Eg: A `winning_move` and/or a `draw_move` variable as a terminating variable will do.

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Well done @mohapakram and @gurbuzertunga  👏 
This milestone is coming along nicely. 💪 

### Status: Changes Required ♻️ 
One more change is required and is shown below.

Please request for another review after implementing the change.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Review by adaorachi — APPROVED** [review]
### Hi @mohapakram  and @gurbuzertunga :wave: 

You have done a good job of implementing the changes requested. You've now met all the minimum requirements for this project. Thumbs up! :+1: 
You can now merge this branch to the base branch of this project while you continue working on the next milestone.
Good luck with it.

---

### STATUS: APPROVED :100: :1st_place_medal: 

---

Happy Coding :relaxed: 

MaryAnn Chukwuka.

**Pull request merged**